cfme/tests/configure/test_docs.py

```diff
@@ -108,7 +109,8 @@ def test_contents(appliance, soft_assert):
         else:
             expected.append('cloudforms')
             assert cur_ver.product_version() is not None
-            expected.append(cur_ver.product_version())
+            if not BZ(1723813, forced_streams=["5.11"]).blocks:
```

Comment:
Q: do you think we have to block this test with BZ for 511 than this workaround.

Reply:
Agreed, I think this test should be blocked in its entirety, not just this conditional statement.

Additionally, `forced_streams` is not necessary here, as the BZ blocks against 5.11 without it.

```
In [1]: from cfme.utils.blockers import BZ
BZ
In [2]: BZ(1723813).blocks
Out[2]: True

In [3]: from cfme.fixtures.pytest_store import store

In [4]: store.current_appliance.version
Out[4]: Version('5.11.0.10')
```

Reply:
Every other assertion is working fine for 5.11, except that the docs on 5.11 do not have expected product version.